main.nf

```diff
@@ -174,6 +175,40 @@ if (params.verbose) mappedBam = mappedBam.view {
   File  : [${it[4].fileName}]"
 }
 
+process RunBamQCmapped {
+  tag {idPatient + "-" + idSample}
+
+  publishDir directoryMap.bamQC, mode: params.publishDirMode
+
+  input:
+    set idPatient, status, idSample, idRun, file(bam) from mappedBamForQC
+
+  output:
+    file(idSample) into bamQCmappedReport
+
+  when: !params.noReports && !params.noBAMQC
+
+  script:
+  """
+  qualimap --java-mem-size=${task.memory.toGiga()}G \
+  bamqc \
+  -bam ${bam} \
+  --paint-chromosome-limits \
+  --genome-gc-distr HUMAN \
+  -nt 16 \
```

Comment:
Hardcode nr of threads?

Reply:
:man_facepalming:

Reply:
It is now corrected...
Thanks for noticing that